packages/fedify/src/webfinger/jrd.ts

```diff
@@ -57,3 +57,52 @@ export interface Link {
    */
   properties?: Record<string, string>;
 }
+
+/**
+ * References a link. See also
+ * [OStatus 1.0 Draft 2](https://www.w3.org/community/ostatus/wiki/images/9/93/OStatus_1.0_Draft_2.pdf)
+ */
+export interface OStatusSubscribeLink {
+  rel: "http://ostatus.org/schema/1.0/subscribe";
+  /**
+   * A URI template (RFC 6570) that can be used to construct URIs by
+   * substituting variables. Used primarily for subscription endpoints
+   * where parameters like account URIs need to be dynamically inserted.
+   */
+  template: string;
+}
```

Comment:
Instead of declaring a separate `OStatusSubscribeLink` type, how about adding a `template` field to the existing `Link` type? I think the `template` field might be useful even when the `rel` field is not `"http://ostatus.org/schema/1.0/subscribe"`.